cmd/machine-controller/main.go

```diff
@@ -166,6 +167,7 @@ func main() {
 	flag.StringVar(&nodeRegistryMirrors, "node-registry-mirrors", "", "Comma separated list of Docker image mirrors")
 	flag.StringVar(&nodePauseImage, "node-pause-image", "", "Image for the pause container including tag. If not set, the kubelet default will be used: https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/")
 	flag.StringVar(&nodeHyperkubeImage, "node-hyperkube-image", "k8s.gcr.io/hyperkube-amd64", "Image for the hyperkube container excluding tag.")
+	flag.StringVar(&nodeKubeletImage, "node-kubelet-image", "quay.io/poseidon/kubelet", "Image for the kubelet container excluding tag.")
```

Comment:
We should probably add to the description that Hyperkube is used only for < 1.18 and Kubelet for >= 1.18.

Reply:
There is such comment in the updated doc

Reply:
I think having it in both places would be useful, but I'll leave it up to you and @irozzo-1A for the final decision.